app/static/admin/js/pages/analytics/graph/analytics-line-graph.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+define(["../analytics-util", "./analytics-graph"], function(util, Graph) {
+
+  function LineGraph(elementId, keyName, valueName, startDate, endDate, color) {
+    Graph.call(this, elementId, keyName, valueName, startDate, endDate, color);
+  }
+
+  LineGraph.prototype = Object.create(Graph.prototype);
+  LineGraph.prototype.constructor = LineGraph;
+
+  LineGraph.prototype.updateChart = function(startDate, endDate) {
+    var that = this;
```

Comment:
Typically people would call these `var self = this` rather than `that`. No need for change. just a fyi.